recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ test:
     - docformatter 1.7.*
     - isort 5.12.*
     - jq 1.6.*
+    - make >=3.8
```

Comment:
A developer needs to run `make` in a development shell for e.g. `make format` and `make test` and, in case their OS does not provide `make`, `uwtools` should provide its own. This line (and the similar line in the derived `meta.json` above) ensures that.